src/Controller/Component/ApiFormatterComponent.php

```diff
@@ -90,6 +90,13 @@ protected function addIncluded(array $resource, Collection $included): array
      */
     protected function extractFromIncluded(Collection $included, array $relationshipData): array
     {
+        // include=object case
```

Comment:
[super minor] case is 1-1 relationship - `object` relation in translations is a special case